ServerManager/api/editGameSession.php

```diff
@@ -38,6 +38,7 @@
 // alternative to recreate function in GameSession class
 $gamesession->edit();
 if ($action == 'recreate') {
+    // GameListCreationMessageHandler will dispatch the GameSaveLoadMessage if this is actually a save reload instead
```

Comment:
ah so after the handling of gamelist create , there will be a dispatch of gameSaveload?